bootstrap.py

```diff
@@ -270,6 +273,16 @@ def enable_rhsmcertd():
     exec_service("rhsmcertd", "restart")
 
 
+def is_registered():
+    # Check if all required certificates are in place (i.e. a system is
+    # registered to begin with) before we start changing things
+    if (os.path.exists('/etc/rhsm/ca/katello-server-ca.pem') and
```

Comment:
I'd do just `return os.path.exists('/etc/rhsm/ca/katello-server-ca.pem') and os.path.exists('/etc/pki/consumer/cert.pem'))`

Reply:
Ya, that's a nicer, more Pythonic way of doing it. Fixed.